apps/prairielearn/src/components/Scorebar.html.ts

```diff
@@ -9,19 +9,22 @@ export function Scorebar(
   }: { minWidth?: string; maxWidth?: string; classes?: string } = {},
 ) {
   if (score == null) return '';
+
+  const percentage = `${Math.floor(score * 100) / 100}%`;
```

Comment:
JavaScript actually has a special function for this, which also guarantees that there will be no issues with floating point representations. I'd also change the names here for clarity:
* Change the argument of the function from `score` to `scorePerc`
* Use a name for the string that indicates how it differs from the numeric value, e.g., `scorePercString`

This gives:
```suggestion
  const scorePercString = `${scorePerc.toFixed(2)}%`;
```

Reply:
Doesn't toFixed() round up or down but we only want to round down?

Eg. `${scorePerc.toFixed(1)}%` rounds to 11.3 not 11.2

![image](https://github.com/user-attachments/assets/2b0944cc-94af-4a14-a84c-9e6a0a487238)

Reply:
I wouldn't use toFixed because it forces zeros to the right (54.00). You can use formatPoints instead.

Reply:
https://github.com/PrairieLearn/PrairieLearn/blob/dcfa464e261a0847dfc3803dae180e790152a6d9/apps/prairielearn/src/lib/format.ts#L9

Reply:
> Doesn't toFixed() round up or down but we only want to round down?

Excellent point, I thought about this and I meant to leave in the `* 100 / 100` to handle this, but then I forgot!

But @jonatanschroeder's suggestion of `formatPoints()` is of course the correct answer here.